### PR TITLE
Picking up mice was easier than I though...

### PR DESCRIPTION
**Pull request by taukausanake**
but took longer than it should have for me to figure it out.
You can now pick up mice. Added basically the same things diona use for this feature.
:cl:
add: Mice can now be picked up like diona
/:cl:

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
can glorious tajarans still devour the vermin though?

**Comment by taukausanake** [comment]
Pretty sure I don't even know where that code is. It shouldn't conflict I think?

**Comment by TheDZD** [comment]
For some reason your Paradise.dme is registering 1924 additions and deletions, going to want to fix that.

**Comment by taukausanake** [comment]
WHY DOES IT KEEP DOING THAT?

**Comment by taukausanake** [comment]
Oh another thing. I am technically still using BYOND 509. Should I update to 510 and recompile?

**Comment by Tastyfish** [comment]
No, the master branch is 509.

**Comment by Fox-McCloud** [comment]
Please. Stop. Using. Old. Style. Changelogs.

**Comment by taukausanake** [comment]
@Fox-McCloud  I don't know what other way there is

**Comment by TheDZD** [comment]
Delete the changelog file from the PR, then add this to the top (or bottom, or anywhere else really) of the PR, and remove the extra space inside of the `:cl:` bits.

: cl:
add: adds a thing
fix: fixes a thing
remove: removes a thing
tweak: tweaks a thing
/: cl:

File: code/modules/mob/holder.dm [hunk 1]
```diff
@@ -62,6 +62,7 @@
 	var/obj/item/weapon/holder/H = new holder_type(loc)
 	src.forceMove(H)
 	H.name = name
+	H.icon_state = icon_state
```
Comment (tigercat2000) [inline code/modules/mob/holder.dm]: 91.5% sure this will break aurora's pAI holders @Aurorablade

**Comment by taukausanake** [comment]
Late reply is late. Hold on...

File: code/modules/mob/holder.dm (hunk 1 above)
Comment (taukausanake) [inline code/modules/mob/holder.dm]: Ew. I didn't think about others actually. Was kind of worried about mice not having their color icon move over

**Comment by TheDZD** [comment]
You still left in an old-style changelog in the PR. The .yml file should be nuked entirely, they're not needed at all, the PR description changelogging does the exact same thing, and is generally nicer for people to read through.

**Comment by tigercat2000** [comment]
@TheDZD you realize this means I am going to only use yml changelogs for the rest of time right

**Comment by taukausanake** [comment]
I don't know anything no one told me about this I was just doing what I think was done in the other PRs don't be mean to me. ;_;

Shall I wait for someone else to nuke the .yml files or should I listen to @tigercat2000 ?

**Comment by TheDZD** [comment]
@tigercat2000 :chicken: :rooster:

@taukausanake You have to delete the .yml file from the PR branch entirely, and push the change. After that, you can add an in-line changelog to the PR description. The .yml file was what we used before in-line changelogs.

**Comment by taukausanake** [comment]
@TheDZD Okay, so I did a diff on my paradise.dme and the one on the repository. It reported that they are exactly alike. Also 1924 lines in the .dme so it thinks I delete and then re-added them all in. It hasn't changed yet GitHub thinks they have. Am I seriously the only one that has this problem?

**Comment by FalseIncarnate** [comment]
@taukausanake might want to make sure you have "Automatically set FILE_DIR for sub-directories" unchecked. This is found by opening dreammaker, going to Build > "Preferences for paradise..."

That might not be responsible for the .dme deal, but worth checking on anyways as it SHOULD be disabled.

**Comment by Crazylemon64** [comment]
You may want to rename this PR so it can be easily be understood what is done here

**Comment by taukausanake** [comment]
@FalseIncarnate FILE_DIR is unchecked so it's not that at fault.
@Crazylemon64 Yeah... I just kept the defaults of my commit. I'll change it

**Comment by taukausanake** [comment]
Okay, this SHOULD work without having any sort of conflict with pAI icons. On the downside, it might be snowflake code but that doesn't matter to me. What does is that it works with different colored mice so I'm happy.
Can I get this reviewed now? @Fox-McCloud or @TheDZD Whoever is in charge of that stuff?

**Comment by Fox-McCloud** [comment]
your paradise.dme is still screwed as hell.

**Comment by taukausanake** [comment]
It is literally the same file it's supposed to be, just everything being deleted and replaced in the exact same format. I don't know why this happens, it's the same deal in my other PR. I wouldn't be surprised if it happens in another PR.

Any thoughts other than what False mentioned?

**Comment by tigercat2000** [comment]
@taukausanake check the encoding and line endings

**Comment by taukausanake** [comment]
This should fixed my .dme file. And by fix I mean use the original one from the beginning

**Comment by tigercat2000** [comment]
@taukausanake yeah it was the line endings
somehow your git client failed to convert to LF over CRLF

**Comment by taukausanake** [comment]
@TheDZD I'm using Git-Scm on Windows. Is there a way to fix this? I used to use Github but then I realized it's fuck and I liked this terminal better

**Comment by tigercat2000** [comment]
The git terminal normally will convert CRLF files to LF files before pushing. Not sure why it wouldn't. 
@taukausanake try `git config --global core.autocrlf true`

**Comment by taukausanake** [comment]
It was false. Set it to true now

**Comment by taukausanake** [comment]
Yaaaaaaaaaay

**Pull request merged**